Test/SearchTaskTest.cs

```diff
@@ -241,5 +242,53 @@ public static void PrunedDbWithContaminantsTest()
             Assert.That(File.ReadAllLines(Path.Combine(folderPath, @"DbForPrunedDbPruned.xml")).Length > 0);
             Directory.Delete(folderPath, true);
         }
+
+        /// <summary>
+        /// Test ensures pruned databases are written when contaminant DB is searched
+        /// </summary>
+        [Test]
+        public static void PeptideFDRTest()
+        {
+            SearchTask searchTask = new SearchTask()
+            {
+                SearchParameters = new SearchParameters
+                {
+                    WritePrunedDatabase = true
```

Comment:
can just use default params; a bit confusing to specify thing you aren't testing

Reply:
done